python/paddle/nn/functional/pooling.py

```diff
@@ -146,99 +168,69 @@ def avg_pool1d(x,
                count_include_pad=True,
                ceil_mode=False,
                name=None):
```

Comment:
missing data format?

Reply:
It supposes to add in this PR https://github.com/PaddlePaddle/Paddle/pull/26331, hmm I suggest refining it in the next PR